php-frontend/src/main/java/org/sonar/php/parser/LexicalConstant.java

```diff
@@ -100,16 +100,11 @@
   private static final String EXECUTION_OPERATOR = "`[^`]*+`";
 
   /**
-   * Heredoc
+   * Heredoc / Nowdoc
    */
-  public static final String HEREDOC = "(?s)"
-    + "<<<[" + WHITESPACE + "]*(\"?([^\r\n'\"]++)\"?)(?:\\r\\n?+|\\n)(.*?)(?:\\r\\n?+|\\n)\\2";
-
-  /**
-   * Nowdoc
-   */
-  public static final String NOWDOC = "(?s)"
-    + "<<<[" + WHITESPACE + "]*'([^\r\n'\"]++)'.*?(?:\\r\\n?+|\\n)\\1";
+  public static final String NEW_LINE = "(?:\r\n?+|\n)";
+  public static final String HEREDOC = "(?s)(<<<[\u0020\t\u000B\f]*\"?([^\r\n'\"]++)\"?" + NEW_LINE + ")(?:(.*?)" + NEW_LINE + ")?\\2";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this Unicode escape sequence and use the character instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2973)